config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
I don't think we should be auto-refreshing on error. I'm just thinking a refresh should probably always be available, as a manual user-initiated action, somehow. Otherwise there is no escape hatch if our app gets into a bad state that can only be fixed by a complete reset. Maybe this is unnecessarily paranoid, but I have seen some web-view apps that stash a "Reload" option in their ••• menu or somewhere unobtrusive.